py/server/deephaven/parquet.py

```diff
@@ -11,17 +11,16 @@
 import jpy
 
 from deephaven import DHError
-from deephaven._wrapper import JObjectWrapper
 from deephaven.column import Column
 from deephaven.dtypes import DType
 from deephaven.table import Table
+from deephaven.experimental import s3
```

Comment:
This is an interesting case.  We have experimental bleeding into our normal API.  I may be ok with this, but @jmao-denver should weigh in.  I may be ok with it because it is obvious it is experimental, so yanking the rug later could be ok.

Do you expect the API to change going forward?  If not, it could be promoted out of experimental.  If it needs baking time, leave it there.

Reply:
It does need more baking time, especially since we will be adding more capabilities to it very soon. Right now,we only support reading a single parquet file. Going forward, we want to support reading partitioned data, metadata files, etc.

Reply:
If we allow experimental feature to bleed into the normal API,  maybe a factory/helper function `s3_instructions(...)` is more desirable than  forcing the user to use  `experimenta.s3.S3Instructions` directly? At least, when we decide to graduate `S3Instructions` from experimental, the user doesn't need to change the code, even though code change comes with the territory of using experimental features.